IOSSecuritySuite/FishHookChecker.swift

```diff
@@ -103,18 +104,18 @@ internal class FishHookChecker {
         let linkeditCmdName = SEG_LINKEDIT.data(using: String.Encoding.utf8)!.map({ $0 })
```

Comment:
maybe `guard let linkeditCmdName = SEG_LINKEDIT.data(using: String.Encoding.utf8)?.map({ $0 }) else { return }`
is better